game/scripts/npc/abilities/charger/boss_charger_charge.txt

```diff
@@ -56,15 +56,15 @@
       "07"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "hero_pillar_damage"                              "8000 10000 12000"
+        "hero_pillar_damage"                              "7000 8000 9000"
```

Comment:
Revert this. I didnt see anyone die from this (even new players).

Reply:
I rescaled it to "8000 9000 10000" I have definetly seen charger get multiple kills.

Reply:
Also I have increased the glancing damage. Meaning if you get hit by the glances a few times a direct pin will probably kill you.